candi/bashible/bashible.sh.tpl

```diff
@@ -253,6 +253,16 @@ function main() {
     else
       REBOOT_ANNOTATION=null
   fi
+ if [ "$FIRST_BASHIBLE_RUN" != "yes" ]; then
+    while true; do
+      if kubectl_exec label nodes $(bb-d8-node-name) node.deckhouse.io/bashible-first-run-finished=true; then
```

Comment:
We decide, that we need check file existing before add label
It needs for one shot label adding

If file exists do not doing anymore, if does not existing should to add label

file can be named, for example, /var/lib/bashible/bashible-fully-initialized